mev_inspect/classifiers/specs/zero_ex.py

```diff
@@ -1,9 +1,78 @@
+from typing import Optional, List
+from mev_inspect.schemas.transfers import Transfer
+from mev_inspect.schemas.swaps import Swap
 from mev_inspect.schemas.traces import (
+    DecodedCallTrace,
     Protocol,
 )
 from mev_inspect.schemas.classifiers import (
     ClassifierSpec,
+    SwapClassifier,
 )
+from mev_inspect.classifiers.helpers import (
+    get_amount_transferred_to_address,
+    get_amount_transferred_by_token_address,
+)
+
+
+class ZeroExSwapClassifier(SwapClassifier):
+    @staticmethod
+    def parse_swap(
+        trace: DecodedCallTrace,
+        prior_transfers: List[Transfer],
+        child_transfers: List[Transfer],
+    ) -> Optional[Swap]:
+
+        token_in_amount: int = 0
+        taker_address: str = ""
+
+        order: List = trace.inputs["order"]
+
+        token_in_address: str = order[0]
+        token_out_address: str = order[1]
+
+        if "Rfq" in trace.function_signature:
```

Comment:
I think better to be explicit and use the full signature. If 0x adds a new `Rfq` function with different ordering, this silently breaks

ex:
```
RFQ_SIGNATURES = [
"fillRfqOrder((address,address,uint128,uint128,address,address,address,bytes32,uint64,uint256),(uint8,uint8,bytes32,bytes32),uint128)",
    ....
]

if trace.function_signature in RFQ_SIGNATURES:
    ...
```

Reply:
also functional kind of does the check on the len of order because you know the signature has that many parameters

right now we're assuming the len of order is valid if Rqf is in